ReachabilitySwift.podspec

```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
     :tag => 'v'+s.version.to_s
   }
   s.source_files = 'Sources/Reachability.swift'
-  s.resource_bundles = {"Kingfisher" => ["Sources/PrivacyInfo.xcprivacy"]}
+  s.resource_bundles = {"ReachabilitySwift" => ["Sources/PrivacyInfo.xcprivacy"]}
```

Comment:
Does this need to match the `.name` or the `.module_name` property? 
Should it just be `{"Reachability" => ["Sources/PrivacyInfo.xcprivacy"]}`

Reply:
I think no matter how, the name should be renamed it back to the package.

Reply:
> Does this need to match the `.name` or the `.module_name` property? Should it just be `{"Reachability" => ["Sources/PrivacyInfo.xcprivacy"]}`

I tested both case with archiving an example project:

```
Example.xcarchive/Products/Applications/Example.app/Frameworks/Reachability.framework/Reachability.bundle/PrivacyInfo.xcprivacy

Example.xcarchive/Products/Applications/Example.app/Frameworks/Reachability.framework/ReachabilitySwift.bundle/PrivacyInfo.xcprivacy
```

Using XCode's Generate Privacy report feature, both cases generate the report correctly and include the library's privacy information. (Need to make some changes in reference to #406).

It doesn't look like we need to match any of them. More importantly, the existing `Kingfisher` naming would conflict with the real [Kingfisher](https://github.com/onevcat/Kingfisher) library if I also add that dependency:

![image](https://github.com/ashleymills/Reachability.swift/assets/16165094/efe93868-a87c-4d0f-9ee6-6a58831444d0)

So I think ReachabilitySwift would be a better choice, which avoids potential conflict issues with [tonymillion/Reachability](https://github.com/tonymillion/Reachability).

@SethBaldwin What do you think?

Reply:
Yeah, that makes sense. Thanks for testing!